### PR TITLE
NP-255

### PR DESCRIPTION
**Pull request by AlessandroCinque**
https://jira.tools.tax.service.gov.uk/browse/NP-255

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -1,19 +1,16 @@
-import play.core.PlayVersion
-import play.sbt.PlayImport._
-import sbt.Keys.libraryDependencies
-import sbt._
+import sbt.*
 
 object AppDependencies {
 
-  private val bootstrapVersion = "9.0.0"
+  private val bootstrapVersion = "9.1.0"
```
Comment (Reece-Carruthers) [inline project/AppDependencies.scala:5]: Should this not be 9.2.0 going off the catalogue?

**Review by TheDanielDoyle — DISMISSED** [review]

File: project/AppDependencies.scala (hunk 1 above)
Comment (AlessandroCinque) [reply project/AppDependencies.scala:5]: My eyes have truly deceit me. I was checking the catalogue and I could swear I saw 9.1.0 , thanks for noticing

Comment (Reece-Carruthers) [reply project/AppDependencies.scala:5]: :laughing: it might have just updated

**Review by Reece-Carruthers — APPROVED** [review]

File: project/AppDependencies.scala (hunk 1 above)
Comment (AlessandroCinque) [reply project/AppDependencies.scala]: Either that or I need a jug of expresso

**Pull request merged**